src/environment/stella_environment.cpp

```diff
@@ -178,6 +178,12 @@ void StellaEnvironment::soft_reset(){
   m_state.incrementFrame();
 }
 
+/** Emulates a NOOP action for both players */
+void StellaEnvironment::wait()
```

Comment:
How hard is it to deal with it in the game instead of dealing with it in the ALE?